test/verify/test_acosh.cpp

```diff
@@ -23,20 +23,23 @@
  */
 
 #include "verify_program.hpp"
+#include <migraphx/literal.hpp>
 #include <migraphx/program.hpp>
 #include <migraphx/generate.hpp>
 #include <migraphx/make_op.hpp>
 
-struct test_acosh : verify_program<test_acosh>
+template <typename CType>
+struct test_acosh : verify_program<test_acosh<CType>>
 {
     migraphx::program create_program() const
     {
         migraphx::program p;
-        auto* mm = p.get_main_module();
-        migraphx::shape s{migraphx::shape::float_type, {16}};
+        auto* mm                      = p.get_main_module();
+        migraphx::shape::type_t dtype = migraphx::shape::get_type<CType>();
+        migraphx::shape s{dtype, {16}};
         auto x       = mm->add_parameter("x", s);
-        auto min_val = mm->add_literal(1.1f);
-        auto max_val = mm->add_literal(100.0f);
+        auto min_val = mm->add_literal(migraphx::literal{migraphx::shape{dtype}, {1.1}});
+        auto max_val = mm->add_literal(migraphx::literal{migraphx::shape{dtype}, {100.0}});
```

Comment:
There a reason why you dropped the  .f here vs some of the other test? Does this need to be converted from double->fp8?

Reply:
No reason.  I probably removed it because literal also had dtype associated. therefore it will get interpreted based on it. 
added back.